app/models/word.rb

```diff
@@ -1,2 +1,4 @@
 class Word < ApplicationRecord
+  belongs_to :lesson
```

Comment:
Mối quan hệ giữa `word` và `lesson` là quan hệ nhiều - nhiều (n - m), thông qua bản lesson_word. 
Giống như kiểu 1 từ này có thể tồn tại trong nhiều bài học khác nhau, và 1 bài học thì có nhiều từ vậy đó.
Em cần xem lại trong rails guide ở đây https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association